dockers/docker-database/Dockerfile.j2

```diff
@@ -47,6 +47,8 @@ RUN apt-get clean -y                                  && \
              s/^# syslog-enabled no$/syslog-enabled no/; \
              s/^# unixsocket/unixsocket/;                \
              s/redis-server.sock/redis.sock/g;           \
+             s/^# io-threads/io-threads/;                \
```

Comment:
>io-threads [](http://example.com/codeflow?start=18&length=10)

The pattern could capture lots of lines, like below one. Suggest add `$` at the end.

Reply:
Improved by using regex matching